spec/controllers/questions/mailing_address_controller_spec.rb

```diff
@@ -25,13 +16,6 @@
       expect(response).to be_successful
     end
 
-    context "when mailing address on the intake is not present" do
-      it "pre-fills mailing address from the user" do
-        get :edit
-        expect(response.body).to include(user.street_address)
-      end
-    end
-
```

Comment:
I feel like if we are retaining the pre-fill functionality, we should keep the test. I'm okay either including or excluding user-based pre-fill, we should just test if including.